### DISCUSSION TITLE
How to use Nextcloud Talk behind proxy

### DISCUSSION THREAD

**Discussion by lianqi1998** [nextcloud/all-in-one#5517] [Questions] [answered]
Now I can configure AIO to proxy Nextcloud through Nginx, but I found that Nextcloud Talk doesn't work behind the proxy. How should I set it up specifically?

**Comment by szaimen** [comment, marked as answer]
Hi, the talk port (3478) cannot be proxied via a reverse proxy and must be exposed directly.
